packages/validators/src/index.ts

```diff
@@ -7,5 +8,40 @@ export const CreatePostSchema = z.object({
 
 export const UpdateUserSchema = z.object({
   id: z.number().int(),
-  notificationsBanners: z.boolean().optional()
-})
\ No newline at end of file
+  notificationsBanners: z.boolean().optional(),
+})
+
+export const SetSchema = z.object({
+  id: z.string(),
+  type: z.nativeEnum(SetType),
+  reps: z.array(z.number().int()),
+  weight: z.array(z.number()),
+  // exerciseLogId: z.number().int(),
+  exerciseId: z.number().int(),
+  unilateral: z.boolean().default(false),
+})
+
+export const ExerciseSchema = z.object({
+  id: z.number().int(),
+  name: z.string(),
+  note: z.string().optional().nullable(),
+  bodyPart: z.nativeEnum(BodyPart),
+  category: z.nativeEnum(Category),
+  sets: z.array(SetSchema),
+})
+
+export const WorkoutSchema = z.object({
+  id: z.number().int(),
+  name: z.string(),
+  description: z.string().optional(),
+  duration: z.number().int(),
+  finishedAt: z.date(),
+  exercises: z.array(ExerciseSchema),
+  userId: z.number().int(),
+})
+
```

Comment:
How did you make it compatible with

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/7e921eb2-e138-4af7-b847-052204f4adee)